### DISCUSSION TITLE
Todoist Integration

### DISCUSSION THREAD

**Discussion by SpaceNerden** [mgmeyers/obsidian-kanban#571] [Ideas & Feature Requests]
It would be amazing if you can link a todoist project to kanban:

Todoist: 
![image](https://user-images.githubusercontent.com/72929861/176607270-b869fd00-1e4d-4996-b1dc-446742f67c77.png)

---

Obsidian:
![image](https://user-images.githubusercontent.com/72929861/176607327-4d14899e-114e-4a8e-976b-c167b6224f8b.png)

The descriptions are in a note in the card. Not sure what to do about the priorities though.

**Comment by SpaceNerden** [comment]
Also, Todoist has a kanban board already. Forgot to mention that.
![image](https://user-images.githubusercontent.com/72929861/176613699-0258b04c-3b2b-494d-b5e8-d42cf6e3314e.png)

**Reply by lguenth** [reply]
Use a plugin like Todoist Text to convert your tasks to Markdown, add it underneath a heading in your kanban, done.
